### PR TITLE
docs: add eliran eliassy to GDE list

### PR DESCRIPTION
**Pull request by eliraneliassy**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -862,5 +862,13 @@
...
     "bio": "Carlos Caballero is a computer science engineer and a PhD in Computer Science from the University of Málaga. He earned his masters in Software Engineering and Artificial Intelligence. He is a full-time teacher in vocational training. He has written several teaching books and more than a dozen research articles in the first international journals.",
     "groups": ["GDE"]
+  },
+  "eliraneliassy": {
+    "name": "Eliran Eliassy",
+    "picture": "eliraneliassy.png",
+    "twitter": "eliraneliassy",
+    "website": "https://eliassy.dev",
+    "bio": "Eliran is an Experienced FE developer, located in Tel Aviv, Israel. He is a developer, trainer, and consultant with focus on Angular and the eco-system around it. He gives talks and workshops on Angular, RXJS, NGRX and related technologies. As an active part of the community, he is constantly involved in growing it and interacting with the members. Eliran is one of the Organisers of AngularUP Annual Conference in Tel Aviv and also an organizer and leading the Angular meetup & community in Israel. He is often speaking at conferences and Meetups and also writing for AngualrInDepth.com.",
```
Comment (kapunahelewong) [inline aio/content/marketing/contributors.json:871]:
```suggestion
    "bio": "Eliran is an Experienced FE developer, located in Tel Aviv, Israel. He is a developer, trainer, and consultant with a focus on Angular and the eco-system around it. He gives talks and workshops on Angular, RXJS, NGRX, and related technologies. As an active part of the community, he is constantly involved in growing it and interacting with the members. Eliran is one of the Organisers of AngularUP Annual Conference in Tel Aviv and also an organizer and leading the Angular meetup & community in Israel. He often speaks at conferences and meetups and also writes for AngualrInDepth.com.",
```

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Congratulations, @eliraneliassy!!! I just have a few optional suggestions for the text. 

Regarding your awesome photo, could you try to get it down to under 30kb?

Btw, we will also need marketing review. Thank you!

**Comment by eliraneliassy** [comment]
Hi @kapunahelewong !
Should be alright now. Thanks!

**Review by kapunahelewong — APPROVED** [review]
LGTM! Congrats, @eliraneliassy 🌟

Reviewed-for: global-docs-approvers

**Comment by petebacondarwin** [comment]
@eliraneliassy - please can you rebase to resolve the conflicts, then we can get this merged.

**Comment by eliraneliassy** [comment]
@petebacondarwin Done.

**Comment by petebacondarwin** [comment]
The failure in the CI is due to a problem that will be fixed by https://github.com/angular/angular/pull/40184, which will not land until the New Year now. And in any case neither will your PR. So please hold tight and rebase once that PR lands in master.

**Comment by eliraneliassy** [comment]
@petebacondarwin @kapunahelewong 
I see that #40184 is closed.
I'm Resolving conflicts here.
Can you re-check, please?

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Just a small fix required: 

>  Error: The following keys in 'contributors.json' are not in alphabetical order: 'yurzui' and 'eliraneliassy'.

**Comment by eliraneliassy** [comment]
@petebacondarwin oops...Fixed :)

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**